tests/pytests/test_json.py

```diff
@@ -1010,10 +1010,15 @@ def testRedisCommands(env):
     env.expect('ft.search', 'idx', 'ri*', 'NOCONTENT').equal([1, 'doc:1'])
 
     # Test Redis COPY
-    env.execute_command('COPY', 'doc:1', 'doc:2')
-    env.execute_command('COPY', 'doc:2', 'dos:3')
+    if not server_version_at_least(env, "6.2.0"):
+        env.execute_command('JSON.SET', 'doc:2', '$', r'{"t":"riceratops","n":"9072","flt":97.2}')
+        env.execute_command('JSON.SET', 'dos:3', '$', r'{"t":"riceratops","n":"9072","flt":97.2}')
+    else:
+        env.execute_command('COPY', 'doc:1', 'doc:2')
+        env.execute_command('COPY', 'doc:2', 'dos:3')
```

Comment:
A bit nitpicking but consider swapping (simpler and the `if` block is more likely to occur)
Not mandatory.
```suggestion
    if server_version_at_least(env, "6.2.0"):
        env.execute_command('COPY', 'doc:1', 'doc:2')
        env.execute_command('COPY', 'doc:2', 'dos:3')
    else:
        env.execute_command('JSON.SET', 'doc:2', '$', r'{"t":"riceratops","n":"9072","flt":97.2}')
        env.execute_command('JSON.SET', 'dos:3', '$', r'{"t":"riceratops","n":"9072","flt":97.2}')
    
```